Makefile

```
@@ -1,4 +1,27 @@
+.PHONY: help
+help:
+	@echo Call one of the available targets:
+	@sed -n 's/\(^[^.#[:space:]A-Z]*\):.*$$/\1/p' Makefile | uniq
+
 .PHONY: update-deps
 update-deps:
 	tools/update-deps --cpanfile cpanfile
 
+.PHONY: test
+test: test-tidy test-critic test-yaml test-author
+
+.PHONY: test-tidy
+test-tidy:
+	tools/tidyall --all --check-only
+
+.PHONY: test-critic
+test-critic:
+	tools/perlcritic --quiet .
+
+.PHONY: test-yaml
+test-yaml:
+	yamllint -c .yamllint --strict ./
```

```suggestion
	yamllint --strict ./
```
https://yamllint.readthedocs.io/en/latest/configuration.html#configuration
> a file named .yamllint, .yamllint.yaml, or .yamllint.yml in the current working directory, or